automation/test.sh

```diff
@@ -499,6 +503,11 @@ if [ -z "$KUBEVIRT_HUGEPAGES_2M" ]; then
   add_to_label_filter '(!requireHugepages2Mi)' '&&'
 fi
 
+# Override, always
+if [[ $TARGET =~ sig-compute-conformance ]]; then
+    label_filter='(sig-compute && conformance)'
```

Comment:
Why is this override required?

Reply:
The conformance set should be able to run without any special hardware and such... So overriding this makes sure we don't include special case...

Reply:
```
# No lane currently supports loading a custom policy
add_to_label_filter '(!CustomSELinux)' '&&'
```
We have this and others that are added, non conditionally

Reply:
Yeah I know but if these are no longer non-conditional - then these should be updated. We shouldn't have to set this twice anyway.

Reply:
So do you prefer to add condition for those non-conditional?

Reply:
I think the non-conditional are still valid for the sig-compute-conformance target? I see no harm in including them in the label filter but maybe I am missing something.

Reply:
They don't harm but I don't want them as I need to make sure the conformance tests will not include  test that would be filtered out by these non-conditionals

Reply:
@brianmcarey Sorry. The "So do you prefer to add condition for those non-conditional?" question still remains

Reply:
> They don't harm but I don't want them as I need to make sure the conformance tests will not include test that would be filtered out by these non-conditionals

Do ye see many of the conformance tests overlapping with these non-conditionals? I don't but I understand your concern. 

The override is probably acceptable as the simplest path forward. 

What kind of conditions were you thinking to apply to these label filters?